source/viz.md

```diff
@@ -85,7 +85,7 @@ As with most coding tasks, it is totally fine (and quite common) to make
 mistakes and iterate a few times before you find the right visualization for
 your data and question. There are many different kinds of plotting
 graphics available to use (see Chapter 5 of *Fundamentals of Data Visualization* {cite:p}`wilkeviz` for a directory).
-The types of plot that we introduce in this book are shown in {numref}`plot_sketches`;
+The types of plots that we introduce in this book are shown in {numref}`plot_sketches`;
```

Comment:
the original text here is grammatically correct -- should be "types of plot" not "types of plots"

Reply:
Hmm, wouldn't this be similar to saying something like "The types of friends I’ve made in this college are great." where both are plural? Or the "The types of apples that I like are red." Both from this article https://grammarhow.com/type-of-types-of-followed-by-singular-or-plural/. This other article suggest that it might be a difference between American and British English https://english.stackexchange.com/a/5546, with the "type of plurals" being AmE which is what we use in the rest of the book. (I'm realizing I'm ESL and might be missing something but got interested and had to look it up =p )

Reply:
honestly it's so minor that I don't mind either way. 

I don't know the actual rule, and I was just parsing it logically: "types of plot" means multiple kinds of plot. "types of plots" means multiple kinds of...multiple plots?